rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rental << self
+    @person = person
+    person.rental << self
```

Comment:
### Optional suggestions

- [ ] This has to be `book.rentals << self` and `person.rentals << self`